src/librustc_mir/transform/mod.rs

```diff
@@ -125,6 +125,39 @@ pub fn default_name<T: ?Sized>() -> Cow<'static, str> {
     if let Some(tail) = name.rfind(':') { Cow::from(&name[tail + 1..]) } else { Cow::from(name) }
 }
 
+/// The `-Zmir-opt-level` an optimization `MirPass` runs at.
+pub enum MirOptimizationLevel {
+    /// A stable optimization which is enabled by default and runs at `mir-opt-level` > 0.
+    Stable,
+    /// An unstable optimization which is disabled by default and only runs at `mir-opt-leel` > 1.
+    /// Generally, these optimizations should work but are disabled because of performance reasons
+    /// or because they have a few known issues.
```

Comment:
Please add "only runs when mir-opt-level > 1".

Reply:
Isn't that what it says on line 132 (except for the typo)?

Reply:
Oops :)